src/Functions.php

```diff
@@ -57,10 +57,15 @@ function beforeEach(Closure $closure = null): BeforeEachCall
     /**
      * Registers the given dataset.
      *
-     * @param  Closure|iterable<int|string, mixed>  $dataset
+     * @param  Closure|iterable<int|string, mixed>|null  $dataset
+     * @return array|void
      */
-    function dataset(string $name, Closure|iterable $dataset): void
+    function dataset(string $name, Closure|iterable $dataset = null)
```

Comment:
Should this still have a return type hint?

```php
function dataset(string $name, Closure|iterable $dataset = null): array|void
```

Reply:
Void can only be used as a standalone type.

Reply:
Of course; my bad.